docs/developers/Instrumentation.md

```diff
@@ -27,11 +27,18 @@ services.Instrumentation.config.statsd {
 
 There is also an additional configuration value that can be set: 
 
-```bash
+```hocon
 # Rate at which Cromwell updates its gauge values (number of workflows running, queued, etc...)
 system.instrumentation-rate = 5 seconds
 ```
 
+If you have multiple Cromwell instances, and would like to separate the instrumentation path for each instance, set the `system.cromwell_id` with the unique identifier for your each Cromwell configuration. For example,
```

Comment:
suggestion: "set `system.cromwell_id` with the unique identifier for each Cromwell instance."

Reply:
👍